test/_certificates/root_ca.pub_key

```diff
@@ -0,0 +1,14 @@
+-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAqo/ZXI+Xfx7U4wX5m3q9
```

Comment:
Suggest extracting the modulus, exponent and keyIdentifier for compatibility with `X509.sol`'s `RSAPublicKey` struct.

Reply:
These information are extracted to the following files, with exception of the modulus that uses the default one:
- test/_certificates/root_ca.pub_key.modulus
-  test/_certificates/root_ca.authority_key

Reply:
Ah, sorry. Didn't spot those.